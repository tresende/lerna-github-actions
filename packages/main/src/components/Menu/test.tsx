import React from 'react'
import { render } from '@testing-library/react'

import Component from '.'
import { BrowserRouter } from 'react-router-dom'

describe('<Menu />', () => {
  it('should ...', () => {
    const { container } = render(
      <BrowserRouter>
        <Component />
      </BrowserRouter>
    )
    expect(1).toBe(1)
    expect(container.firstChild).toMatchSnapshot()
  })
})
