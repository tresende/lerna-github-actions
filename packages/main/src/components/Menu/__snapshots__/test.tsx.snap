// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Menu /> should ... 1`] = `
<nav
  class="sc-bdvvtL joudpd"
>
  <a
    href="/"
  >
    List
  </a>
  <a
    href="/Create"
  >
    Create
  </a>
</nav>
`;
